pkg/api/http_server.go

```diff
@@ -130,7 +130,10 @@ func (hs *HTTPServer) Run(ctx context.Context) error {
 		}
 
 		// Make socket writable by group
-		os.Chmod(setting.SocketPath, 0660)
+		if err := os.Chmod(setting.SocketPath, 0660); err != nil {
+			hs.log.Debug("server was shutdown gracefully")
```

Comment:
Log the error and/or information about failed to make socker writable?

[Line 128](https://github.com/grafana/grafana/pull/19657/files#diff-06cc78b0d87adc91d3c4b7c27db1b05dR128) above would also be nice to log the actual error since there's no check if error is `http.ErrServerClosed` like is done for HTTP.